README.md

```diff
@@ -29,6 +29,15 @@ to Kafka for each package release to be solved using Thoth [Solver](https://gith
 
 Consumer is currently able to handle the following Kafka messages focused on:
 
+#### Environment Variables
```

Comment:
It should be above `Consumer is currently able to handle the following Kafka messages focused on:` ?

Reply:
Yeah, my bad